gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dnumslider_ttt2.lua

```diff
@@ -144,9 +145,15 @@ function PANEL:SetValue(val)
 
 	self:ValueChanged(self.m_fValue)
 
+	if ignoreConVar then return end
+
 	if self.conVar then
 		self.conVar:SetFloat(self.m_fValue)
 	end
+
+	if self.serverConVar then
+		cvars.ChangeServerConVar(self.serverConVar, tostring(val))
```

Comment:
Generally to make the structure clear to other programmers, that copy the code. Because I didnt know, that convars were saved as strings and such conversions might help. The cvars-library makes the same conversion, so of course this is unnecessary. In case of the checkbox a boolean need to be converted to 1 or 0.
So depends what you think makes sense. I'd leave it inside to have an overall similar structure, but it's redundant.